### DISCUSSION TITLE
How to list all studies through QIDO-RS

### DISCUSSION THREAD

**Discussion by sluijs** [microsoft/dicom-server#893] [Q&A]
Is it possible to list all studies through QIDO-RS?  
```
curl --request GET "http://localhost:8080/studies/" --header "Accept: application/dicom+json" --trace-ascii "trace.txt"
``` 
The above or Postman returns a `204 Not Found`. Thanks!

**Comment by jovinson-ms** [comment]
Hi @sluijs, the URI you provided should return metadata on all studies. How many DICOM files had you stored via STOW request?
